testing/trino-product-tests/src/main/java/io/trino/tests/product/hudi/TestHudiSparkCompatibility.java

```diff
@@ -343,6 +357,31 @@ private void createNonPartitionedTable(String tableName, String tableType)
         onHudi().executeQuery("INSERT INTO default." + tableName + " VALUES (1, 'a1', 20, 1000), (2, 'a2', 40, 2000)");
     }
 
+    private void createNonPartitionedTableWithReplaceCommit(String tableName, String tableType)
+    {
+        onHudi().executeQuery(format(
+                """
+                        CREATE TABLE default.%s (
+                          id bigint,
+                          name string,
+                          price int,
+                          ts bigint)
+                        USING hudi
+                        TBLPROPERTIES (
+                          type = '%s',
+                          primaryKey = 'id',
+                          preCombineField = 'ts',
+                          hoodie.clustering.inline = 'true',
+                          hoodie.clustering.inline.max.commits = '1')
```

Comment:
Are all table properties necessary for testing replace commits?

Reply:
They should work without these, so I did remove everything not related to clustering.